uber/utils.py

```diff
@@ -67,10 +73,12 @@ def check_csrf(csrf_token):
     """
     if csrf_token is None:
         csrf_token = cherrypy.request.headers.get('CSRF-Token')
-    assert csrf_token, 'CSRF token missing'
+    if __debug__:
```

Comment:
That's what the [official python documentation](https://docs.python.org/3/reference/simple_stmts.html#grammar-token-assert_stmt) says for the definition of an `assert` is.